.changeset/little-cars-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools/product-json-to-xlsx': minor
+---
+
+Upgrage exceljs
```

Comment:
hmm, is this PR only about changeset or did the upgrade happened in older PR?

Reply:
Yes upgrade didn't happen because I missed to commit this file so with this commit it will be released